install/ui/src/freeipa/widgets/LoginScreen.js

```diff
@@ -53,6 +53,10 @@ define(['dojo/_base/declare',
 
         expired_msg: "Your session has expired. Please re-login.",
 
+        new_ui_msg: "<i class=\"fa fa-info-circle\"></i> We have launched " +
+            "new UI, <a href='${protocol}//${host}/ipa/modern_ui'>" + 
```

Comment:
`s/modern_ui/modern-ui/`

Reply:
Good point, the URL is prettier this way